satpy/etc/readers/aws_l1b_nc.yaml

```diff
@@ -0,0 +1,425 @@
+reader:
+  name: aws_l1b_nc
+  short_name: AWS L1B RAD NetCDF4
+  long_name: AWS L1B Radiance (NetCDF4)
+  description: Reader for the ESA AWS (Arctic Weather Satellite) Sounder level-1b files in netCDF4.
```

Comment:
Is there a link to online documentation of the format we could use?

Reply:
No, unfortunately not, but there will soon be for EPS-Sterna. Not exactly the same format, but close. I hope that the same code can be used for both, but using different yaml files perhaps

Reply:
These also should be AWS/MWS (or just MWS).